src/mono/System.Private.CoreLib/src/System/Reflection/Emit/RuntimeTypeBuilder.Mono.cs

```diff
@@ -638,10 +638,6 @@ protected override ConstructorBuilder DefineTypeInitializerCore()
                 null);
         }
 
-        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2110:ReflectionToDynamicallyAccessedMembers",
-            Justification = "For instance member internal calls, the linker preserves all fields of the declaring type. " +
```

Comment:
How does the linker know that the Mono unmanaged runtime depends on the RuntimeTypeBuilder fields? Do we need to declare the dependency somewhere?